archive/g/groovy/Fibonacci.groovy

```diff
@@ -0,0 +1,17 @@
+class Fibonacci {
+    static void main(String[] args) {
+        if (args.length < 1 || !args[0].isInteger()) {
+            println 'please provide an integer between 1 and 100'
```

Comment:
Based on discussion, let's just change this line to say "please provide an integer". I don't think we need to set limits.